src/libsync/syncengine.cpp

```diff
@@ -349,7 +349,8 @@ int SyncEngine::treewalkFile( TREE_WALK_FILE *file, bool remote )
         item->_remotePerm = QByteArray(file->remotePerm);
     }
     item->_should_update_etag = item->_should_update_etag || file->should_update_etag;
-
+    item->_hasIgnoredFiles    = file->has_ignored_files > 0 ? true : false;
```

Comment:
file->has_ignored_files is an int coming from the db. item->hasIgnoredFiles however is a bool. This conversion is makes it clear how this value is computed.